### DISCUSSION TITLE
Dataview rendered list item is not clickable

### DISCUSSION THREAD

**Discussion by andrey-jef** [ObsidianPublisher/obsidian-github-publisher#102] [Q&A — Plugin] [answered]
Hi

I'm using your template mkdocs and deploy on Cloudflare Pages.

I have a note contained a dataview query which is rendered as an html table, as you stated in the plugin docs page. 

However, the list items (files) are not clickable. I can't click the filename to go to each published daily notes. Is it expected behavior?

Here is the screencap

![image](https://user-images.githubusercontent.com/81280428/219296858-0e560ce2-b99e-4977-ba4a-287a2f0658bd.jpeg)

**Comment by Mara-Li** [comment, marked as answer]
1. Did you enable the links conversion?
2. Are the file exists in the blog?

**Reply by andrey-jef** [reply]
Aha. So this not the expected behavior. I will dive in to the settings and return to you later.

1. I will recheck the "link conversion" setting 
2. Yes. All files are published, with the `share: true` key in frontmatter

Thank you

**Reply by Mara-Li** [reply]
I mean, did you send them on the blog successfully? 
Because the share: true is good but if you forgot to push x)

**Reply by andrey-jef** [reply]
Haha. Yes. The files are sent to the repo successfully. I can see each daily notes on the published website. I also see them in the github repo

**Reply by andrey-jef** [reply]
Fast update until now. The files listed from dataview query are still not clickable as expected.

1. I have just enabled all 3 options "Internal links", "Convert internal links pointing to unshared files", and "[[Wikilinks]]" in the "Content's conversion" section.
2. The "dataview.md" note containing the dataview query and its relevant files are all pushed successfully into the repo.

I doubt that the error might happen because the template is not updated. I'm reading your document site on how to update the template, but still cannot do it. Keep trying to debug here

**Reply by Mara-Li** [reply]
1. Please send me a link or a copy to your repository so I can check it
2. Could you please describe your problem with updating the template in another discussion?

Thanks.

**Reply by andrey-jef** [reply]
Hi

Just want to update that the dataview table works as expected now. That means the listed files are clickable. 

I didn't change anything, just published new notes, changed the folder structure of daily notes and call the command "refresh and publish new notes" multiple times. So i have no idea what was wrong and which changes make it right...

Ticket closed. Thanks for your help

**Reply by Mara-Li** [reply]
I make some update around the template + the plugin so maybe it's that! I noticed that the blog bug around make link in table

**Comment by Mara-Li** [comment]
Also I'm curious. It is possible to get the converted file of this dataview table? Maybe links are not well parsed
